recipe/meta.yaml

```diff
@@ -14,22 +14,18 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record=record.txt
-  skip: True  # [win32]
   features:
-    - vc9   # [win and py27]
-    - vc14  # [win and (py35 or py36)]
+    - vc{{ vc }}   # [win]
 
 requirements:
-  build:
-    - toolchain  # [unix]
+  host:
     - python
     - setuptools
-    - numpy x.x
-    - numpy >=1.9,<2.0.0
-
+    - numpy
+  build:
+    - {{ compiler('cxx') }}
   run:
     - python
-    - numpy x.x
     - numpy >=1.9,<2.0.0
```

Comment:
this is probably better as

``{{ pin_compatible('numpy') }}``

It'll come out the same on unix, where 1.9 is our base numpy version.  On Win, it'll end up as 1.11, because that's our base on win.  We couldn't do 1.9 on win, because scipy refused to compile with it.

Reply:
Fixed.